recipes/RNAtools/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "RNAtools" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "6e6285c54bba3e492581c3b90b2dd86e1f2c2605c6298bee5280fdf0ce19a39b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
it looks like there might be additional run time requirements:

https://github.com/grice/RNAtools/blob/2bb1cb42920191d841fe06940081450f69af5dbe/requirements.txt